tests/store_test.go

```diff
@@ -40,29 +40,14 @@ type StoreSuite struct {
 func (s *StoreSuite) TestStoreNew() {
 	doc := NewStoreFixture()
 	s.False(doc.IsPersisted())
-	s.Len(doc.ID.String(), 24)
+	s.Len(doc.ID.String(), 36)
```

Comment:
replace with `s.False(doc.ID.IsEmpty())`, the purpose of the test is not checking the length of the UUID (which will always be 36 because it is an array and has all the capacity filled), is to see that is not empty.